core/internal/cltest/cltest.go

```diff
@@ -467,12 +467,12 @@ func NewEthMocks(t testing.TB) *evmMocks.Client {
 }
 
 // Deprecated: use evmtest.NewEthClientMock
-func NewEthClientMock(t mock.TestingT) *evmMocks.Client {
+func NewEthClientMock(t *testing.T) *evmMocks.Client {
 	return evmtest.NewEthClientMock(t)
 }
 
 // Deprecated: use evmtest.NewEthClientMockWithDefaultChain
-func NewEthClientMockWithDefaultChain(t testing.TB) *evmMocks.Client {
+func NewEthClientMockWithDefaultChain(t *testing.T) *evmMocks.Client {
```

Comment:
Was this change from the testing.TB interface to concrete *testing.T required for some reason? Probably nbd but unfortunate.

Reply:
This is done because `evmtest.NewEthClientMock(t)` does not accept `t testing.TB`.

Reply:
I think it will go away after we remove this method:
`// Deprecated: use evmtest.NewEthClientMock`
`func NewEthClientMock(t *testing.T) *evmMocks.Client {`